src/main/scala/battleship/routes/ActorRoutes.scala

```diff
@@ -0,0 +1,63 @@
+package battleship.routes
+
+import akka.actor.ActorRef
+import akka.http.javadsl.model.headers.Server
+import akka.pattern.ask
+import akka.http.scaladsl.server.Directives._
+import akka.http.scaladsl.server.directives.MethodDirectives.get
+import akka.http.scaladsl.server.directives.PathDirectives.path
+import akka.http.scaladsl.server.directives.RouteDirectives.complete
+import battleship.GameActor.{GameNotStartedYet, GameStateRequest, StartGame}
+import battleship.game.Boat
+import battleship.GameActor
+import akka.util.Timeout
+import akka.http.scaladsl.marshallers.sprayjson.SprayJsonSupport
+import akka.http.scaladsl.model.{HttpResponse, RequestTimeoutException, StatusCodes}
+import akka.http.scaladsl.server.ExceptionHandler
+import spray.json.DefaultJsonProtocol
+
+import scala.concurrent.Await
+import scala.concurrent.duration._
+
+/**
+  * Created by m06f947 on 18-8-2017.
+  */
+
+object TestJsonSupport extends DefaultJsonProtocol with SprayJsonSupport {
+  case class Test(size: Int, random: String)
+  implicit val PortofolioFormats = jsonFormat2(Test)
+}
+/*
+actorref geeft een probleem, maar die ga ik uiteindelijk toch niet gebruiken omdat we voor de gamemanageractor gaan

```

Comment:
Eens. Ik wilde echter proberen om een message-class van een actor 1 op 1 te porten naar json in de post. Maar met een actorref erin is dat wat onzinnig. Vandaar dat er nu een test-class is om te kijken hoe het werkt.